### PR TITLE
VIRTS 3201-3203 API v2 Agent Documentation

### PR DESCRIPTION
**Pull request by ddavila54**
## Description

Documentation for API v2 docs (Agent HEAD, GET, PATCH)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Swagger UI Docs working.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: app/api/v2/handlers/agent_api.py [hunk 1]
```diff
@@ -33,9 +33,21 @@ async def get_agents(self, request: web.Request):
         agents = await self.get_all_objects(request)
         return web.json_response(agents)
 
-    @aiohttp_apispec.docs(tags=['agents'])
+    @aiohttp_apispec.docs(tags=['agents'],
+                          summary="Retrieve Agent Information",
```
Comment (bleepbop) [inline app/api/v2/handlers/agent_api.py:37]: Just to make sure this is different from the `get_agents` description, can you change this summary to "Retrieve Agent by paw"?

File: app/api/v2/handlers/agent_api.py [hunk 2]
```diff
@@ -47,14 +59,25 @@ async def create_agent(self, request: web.Request):
...
+                              'description': 'ID of the Agent to update'
+                          }])
     @aiohttp_apispec.request_schema(AgentSchema(partial=True, only=['group',
                                                                     'trusted',
                                                                     'sleep_min',
                                                                     'sleep_max',
                                                                     'watchdog',
                                                                     'pending_contact']))
-    @aiohttp_apispec.response_schema(AgentSchema)
+    @aiohttp_apispec.response_schema(AgentSchema(only=[]), description="Returns JSON response with updated Agent fields")
```
Comment (bleepbop) [inline app/api/v2/handlers/agent_api.py:80]: This `only=[]` can be removed from the  AgentSchema here

**Review by bleepbop — CHANGES_REQUESTED** [review]
Two quick changes and it should be good to go!

**Review by ArtificialErmine — APPROVED** [review]
LGTM

File: app/api/v2/handlers/agent_api.py [hunk 3]
```diff
@@ -33,9 +33,21 @@ async def get_agents(self, request: web.Request):
...
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'paw',
+                              'schema': {'type': 'string'},
+                              'required': 'true',
+                              'description': 'ID of the Agent to retrieve information about'
+                          }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
-    @aiohttp_apispec.response_schema(AgentSchema(partial=True))
+    @aiohttp_apispec.response_schema(AgentSchema(only=[]), description="Returns JSON response with specified Agent "
```
Comment (bleepbop) [inline app/api/v2/handlers/agent_api.py:49]: Sorry missed this before, but this `only=[]` can be removed as well

**Review by bleepbop — APPROVED** [review]

**Review by ArtificialErmine — APPROVED** [review]
LGTM

**Pull request merged**